R/qenv-get_var.R

```diff
@@ -1,5 +1,9 @@
 #' Get object from `qenv`
 #'
+#' @description
+#' `r lifecycle::badge("superseded")` by native \R operators/functions:
```

Comment:
Should we mark this function and `join` as superseded?

Reply:
yes 👍